scheme/proto/src/test/java/cz/o2/proxima/scheme/proto/ProtoMessageValueAccessorTest.java

```diff
@@ -290,4 +300,74 @@ public void testGetDefaultValuesFromStructureField() {
     assertTrue(value.containsKey("repeated_string"));
     assertEquals(2, ((List<?>) value.get("repeated_string")).size());
   }
+
```

Comment:
Do we have test for the nullability?

Reply:
It's part of testValueOfFromMessageWithWrappers where I check `value.isEmpty()` first.